docs/operate/reference/advanced-modules/_index.md

```diff
@@ -46,6 +46,73 @@ Running {{< glossary_tooltip term_id="modular-resource" text="modular resources"
 
 However, if you are unable to use modular resources because you need to host `viam-server` on a non-Linux system or have an issue with compilation, you may need to [implement a custom component and register it on a server configured as a remote](/operate/reference/advanced-modules/custom-components-remotes/) on your machine.
 
+## Package and deploy using Docker
```

Comment:
For both Custom APIs and components as remote parts, we have subpages. I think we should do the same here.